doc/docs.rst

```diff
@@ -63,3 +54,15 @@ For users
 
 - | `Index <theindex.html>`_
   | The generated index.
+
+For Users (Legacy)
+==================
+
```

Comment:
We should add a small warning block here (and potentially on top of the documents themselves)

```rst
.. warning::
   These documents are not maintained and might not correspond to the
   current language semantics. Consult the `manual <manual.html>`_ and
   `standard library <lib.html>`_ for the most up-to-date information.
```